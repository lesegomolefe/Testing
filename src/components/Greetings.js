// src/components/Greetings.js

import React from "react";

function Greetings() {
  return <p>Hello, World!</p>;
}

export default Greetings;
