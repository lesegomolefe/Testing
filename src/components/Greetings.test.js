// src/components/Greetings.test.js

import React from "react";
import renderer from "react-test-renderer";
import Greetings from "./Greetings";

test("renders Greetings component correctly", () => {
  const tree = renderer.create(<Greetings />).toJSON();
  expect(tree).toMatchSnapshot();
});
