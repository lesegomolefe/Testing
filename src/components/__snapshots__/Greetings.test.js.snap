// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Greetings component correctly 1`] = `
<p>
  Hello, World!
</p>
`;
